L 06/17/2021 - 18:12:51: SourceMod error session started
L 06/17/2021 - 18:12:51: Info (map "vsh_chinatown_b3") (file "/home/frenzoid/TF2_FF2/tf/addons/sourcemod/logs/errors_20210617.log")
L 06/17/2021 - 18:12:51: [SM] Exception reported: Invalid database Handle 0 (error: 4)
L 06/17/2021 - 18:12:51: [SM] Blaming: sbpp_sleuth.smx
L 06/17/2021 - 18:12:51: [SM] Call stack trace:
L 06/17/2021 - 18:12:51: [SM]   [0] SQL_TQuery
L 06/17/2021 - 18:12:51: [SM]   [1] Line 170, D:\MegaSync\Projects\Sourcemod\scripting\sbpp_sleuth.sp::OnClientPostAdminCheck
L 06/17/2021 - 18:12:51: [sbpp_comms.smx] Connecting to database failed: [2003]: Can't connect to MySQL server on 'oldbox.cloud' (111)
L 06/17/2021 - 18:19:28: Error log file session closed.
L 06/17/2021 - 18:19:58: SourceMod error session started
L 06/17/2021 - 18:19:58: Info (map "vsh_intrusion_rc2") (file "/home/frenzoid/TF2_FF2/tf/addons/sourcemod/logs/errors_20210617.log")
L 06/17/2021 - 18:19:58: [SM] Exception reported: Invalid database Handle 0 (error: 4)
L 06/17/2021 - 18:19:58: [SM] Blaming: sbpp_sleuth.smx
L 06/17/2021 - 18:19:58: [SM] Call stack trace:
L 06/17/2021 - 18:19:58: [SM]   [0] SQL_TQuery
L 06/17/2021 - 18:19:58: [SM]   [1] Line 170, D:\MegaSync\Projects\Sourcemod\scripting\sbpp_sleuth.sp::OnClientPostAdminCheck
L 06/17/2021 - 18:19:58: [sbpp_comms.smx] Connecting to database failed: [2003]: Can't connect to MySQL server on 'oldbox.cloud' (111)
